packages/devtools-network-console/src/_locales/ps-PS/messages.json

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
Do we need to check in the PS if there is a PS generator? or is the intention that PS generator is run on every checkin that changes strings and will replace this file?

Reply:
We don't, strictly speaking, need to check it in. But, it doesn't hurt.

Reply:
Sounds good! I'm fine checking it in, was just curious.